packages/config/src/globals.ts

```diff
@@ -128,7 +130,7 @@ export const hiddenPrefix = '_#_'
 
 export const clearAll = 'clearAll'
 export const allDbs = 'allDbs'
-export const socketPath = 'http://localhost:4000'
+export const socketPath = hostname.toLowerCase().indexOf('herokuapp') !== -1 ? `https://${hostname}` : 'http://localhost:4000'
```

Comment:
I think this socketPath may be wrong.

If we're not on `Heroku`, the browser may be connecting to server other than `localhost`.  So, on a network without an Internet connection one PC will be hosting Serge, and other network PCs will connect to it.

So, I think the hostname needs to come from the `window.location` 

`localhost` only works for us devs.